lib/shared/bucketing-assembly-script/assembly/types/configBody.ts

```diff
@@ -8,32 +8,52 @@ import {
     isValidString,
     getJSONObjFromJSONOptional,
     getStringFromJSONOptional,
+    getBoolFromJSON
 } from '../helpers/jsonHelpers'
 import { Feature } from './feature'
 import { Audience } from './target'
 
+export class Settings extends JSON.Value {
+    readonly disablePassthroughRollouts: bool
+
+    constructor(settings: JSON.Obj) {
+        super();
+        if (settings.has("disablePassthroughRollouts")) {
+            this.disablePassthroughRollouts = getBoolFromJSON(settings, 'disablePassthroughRollouts')
+        } else {
+            this.disablePassthroughRollouts = false;
+        }
+
+    }
+    stringify(): string {
+        const json = new JSON.Obj()
+        json.set('disablePassthroughRollouts', this.disablePassthroughRollouts)
+        return json.stringify()
+    }
+}
+
 export class PublicProject extends JSON.Value {
-    readonly _id: string
-    readonly key: string
-    readonly a0_organization: string
-    readonly settings: JSON.Obj
+    readonly _id: string;
```

Comment:
Can you remove all the `;` you added?

Reply:
yeah, idk why prettier isnt getting rid of them